apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Trace/TraceDecisionPredicateTest.cs

```diff
@@ -29,10 +28,9 @@ public class TraceDecisionPredicateTest
     {
         private HttpRequest CreateRequest(string path)
         {
-            return new DefaultHttpRequest(new DefaultHttpContext())
-            {
-                Path = path
-            };
+            var request = new DefaultHttpContext().Request;
+            request.Path = path;
+            return request;
```

Comment:
```suggestion
            return DefaultHttpContext { Request = { Path = path } };
```

... and then turn it into an expression-bodied method, but I can't suggest that within GitHub

Reply:
Done